workflow/rules/commons/05_refcon.smk

```diff
@@ -17,7 +17,14 @@ import pandas
 
 localrules: refcon_run_dump_manifest, refcon_cache_manifests
 
-DIR_REFCON = pathlib.Path(config['reference_container_folder'])
+
+USE_REFCON = config['USE_REFCON']
+
+if USE_REFCON:
```

Comment:
this is not ideal, because the two constants are easy to mix up...
Better to define DIR_REFCON after it is known whether or not reference containers should be used in the workflow

Reply:
The code now includes an IF statement to check if a reference container should be even used. Only when "True" further checks, e.g. presence of reference folder and/or if available reference container have been selected will be applied.